macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -89,10 +105,24 @@ class TestButtonViewController: NSViewController, NSMenuDelegate {
 		buttonStatesPopup.target = self
 		buttonStatesPopup.action = #selector(TestButtonViewController.stateChanged)
 
+		widthPopup.addItems(withTitles: widths.keys.sorted())
+		widthPopup.selectItem(withTitle: "natural")
```

Comment:
nit: I would factor out the string to a constant

Reply:
I was always arranging these popups so that the initial selection is also the first menu item, so these 'selectItem' calls were redundant.